content/operate/rs/references/rest-api/api-reference/openapi.json

```
@@ -0,0 +1,243 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Authentication Service REST API",
+    "description": "REST API for the Authentication Service, primarily used for exposing SSO related endpoints.",
```

@sonya-spasova, if there's not a good way to add a work-in-progress banner to the new API reference page, maybe just adding something like this to the description will be good enough? I tested this change in my local env and was able to see the message under the page title.
```suggestion
    "description": "Redis Enterprise Software REST API reference. This API reference is still a work in progress. For more API paths, see the [REST API requests](https://redis.io/docs/latest/operate/rs/references/rest-api/requests/) reference pages.",
```